api-reference/api-reference/relay/relay-retrieve.mdx

```diff
@@ -0,0 +1,3 @@
+---
+openapi: openapi_spec get /relay/{relay_id}
```

Comment:
we'll have to make changes in `mint.json` also, we can test the api-reference changes locally using mintlify - https://mintlify.com/docs/development